File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/DeleteReleaseByFlagKeyExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class DeleteReleaseByFlagKeyExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["ApiKey"] = "YOUR_API_KEY"};

        try
        {
            new ReleasesBetaApi(config).DeleteReleaseByFlagKey(
                projectKey: null,
                flagKey: null
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ReleasesBetaApi#DeleteReleaseByFlagKey: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/DeleteReleaseByFlagKeyExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.ReleasesBetaApi(configuration).deleteReleaseByFlagKey(
  "projectKey_string", // projectKey
  "flagKey_string", // flagKey
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ReleasesBetaApi#deleteReleaseByFlagKey:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
